x-pack/plugins/security_solution/cypress/e2e/exceptions/alerts_table_flow/add_exception.cy.ts

```diff
@@ -67,13 +66,19 @@ describe('Exceptions Table', () => {
       'exceptionListResponse'
     );
 
+    login();
```

Comment:
same question as in `x-pack/plugins/security_solution/cypress/e2e/exceptions/add_edit_flyout/flyout_validation.cy.ts`

Reply:
There is an assertion being ran 2 lines below, this can be refactored.